src/extensions/boosts.py

```diff
@@ -0,0 +1,12 @@
+import interactions as discord
+import main
```

Comment:
`main` is also an unused import here

Reply:
It will now be used for the config dict that jed suggested.